src/main/java/org/group26/ContinuousIntegrationServer.java

```diff
@@ -35,66 +52,77 @@ public void handle(String target,
 
         response.getWriter().println("CI job done");
     }
-    
+    public JSONObject getAPItoJsonOb(HttpURLConnection connn) throws IOException {
```

Comment:
Suggestion: rename function to: `getJsonFromConnection`. It describes the function better.

Also, add a java-doc comment with a brief explanation of its purpose and goal. Additionally, it should state its input parameters and what it returns.